### ISSUE TITLE
plans file

### ISSUE THREAD

**Issue by visoo10** [matsim-org/matsim-code-examples#1061]
Is there any way to upload to simunto via a plans file without having previously uploaded to the matsim GUI?
I´m doing all the code in python and I don´t know how I could do it.

**Comment by mrieser** [comment]
Yes, you can add the plans-file that you would use as input for a simulation to Via. You can then create a layer "Agents – from Population file". 

This should allow you to visualize the activity locations of your population, and you could click on some agents or search for them to look at their plan.

If your plans-file already has routes (e.g. for car trips) (although if you created it in python that's probably not the case as it would require routing on the network), you could also create a layer "Vehicles – from Population". Then you could also see the vehicles according to the planned routes.

Note: Via will NOT run the simulation for you. In the case you use a plans.xml as input, it will only show you what the agents planned to do, no matter how realistic or not those plans are. This functionality should only be used to debug/verify your input plans, but will not replace a proper simulation with MATSim (with or without the GUI).
